modules/http/src/main/java/com/spotify/ffwd/http/HttpClient.java

```diff
@@ -36,7 +36,7 @@
 import okhttp3.Response;
 
 public class HttpClient {
-    private static final String V1_BATCH_ENDPOINT = "v1/batch";
+    private static final String V2_BATCH_ENDPOINT = "v2/batch";
```

Comment:
How are we going to switch all clients of metricsapi that currently sending to "v1/batch" to start using "v2/batch"? I think it will break those clients. I think we should support both end points and in "v1/batch" one convert submitted metrics to new v2 metric.

Reply:
There is no need to switch every client to v2/batch.  Let's look at  the "pipeline" 

httpclient ->httpDecoder -> InputChannel   ---->pubsub.   

Since httpDecoder can support both v1/batch and v2/batch there is no need to switch every client.
Essentially, we use the uri to determine the format of the metric in this case we have to use v2/batch otherwise it won't work.

Reply:
I see. Thank you 👍